### ISSUE TITLE
Including the local library for shape files - with acknowledgements

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4936]
I copy the correspondence from: HDX

From: HDX <noreply@humdata.org> 
Sent: Saturday, September 8, 2018 12:29 PM
To: HDX <hdx.feedback@gmail.com>
Subject: Faq: request from user

 
A user sent the following question using the FAQ contact us form.
Name: Roger Stern
Email: r.d.stern@reading.ac.uk
Section: Sharing and Using Data
Message: We are preparing a front end to the R-statistical system, called R-Instat. This is largely being done in Africa, and is designed for students and researchers in Africa and beyond. Our software (like R) is free and open source. It is to include some mapping tools. We have a data library, within the software and would like to download and include some of your shape files, because some of our potential users will not have easy internet access. My understanding is that this should be OK as long as we have an acknowlegments section in the software that lists the files we have downloaded. Could you confirm that this is OK, assuming we restrict our downloads to the creative commons files in your collection? 

Humanitarian Data Exchange | Centre for Humanitarian Data 
Sign up for our newsletter | Follow us on Twitter | Contact us 

**Reply as follows:** 
Dear Roger,

Thank you for reaching out. Indeed you may use the data in this way (see below) as long as usage of the data is in accordance with the license terms as specified by the data contributor in the dataset license field.

Best wishes,
The HDX Team

Humanitarian Data Exchange
Sign up for our newsletter | Follow us on Twitter | Contact us

Sent on behalf by:
Godfrey Takavarasha
Data Manager
takavarasha@un.org

I therefore suggest that:
a) @dannyparsons and @volloholic confirm the steps below are OK.  If so:
a) The library button be restored on **Climatic > Import Shape Files** dialogue.
b) @JacklineKemboi you had 2 sources.  We check that both are supplying Creative Commons files and then put the sample from Rwanda (and also Kenya) there now.  They go into a special directory and that's where you go automatically, when you press the **Library button** - just like the Open NetCDF files.
c) I draft an **acknowledgements text** to go into the R-Instat Help file.
d) An **acknowledgements** item is added to the bottom of the Help menu that points to this entry in the Help file.

**Comment by dannyparsons** [comment]
I thought that the question was that the Rwanda shapefiles had an **Other** licence and it wasn't clear what that meant. But good to know that other Creative Commons files can be added.

**Comment by rdstern** [comment]
Yes, you are right.  They are "Other".  I looked at what that means in this case and wonder, from that,  whether we just assume Ok and acknowledge for them.

**Comment by dannyparsons** [comment]
I don't think we can assume its ok. We could contact them directly though with a short request about what we want to do.
